src/methods/filterCourses.ts

```diff
@@ -135,27 +121,17 @@ const createModuleSelector = () => {
 }
 
 const applyFilter = (moduleCode: ModuleCode): void => {
-  let viewMode: "list" | "thumbnail"
-  let courses: HTMLElement[]
-
-  const coursesListContainer = document.querySelector(".courselist tbody")
-  const coursesThumbnailContainer = document.querySelector(
-    ".mycourses-body .section"
-  )
+  const [coursesListContainer, coursesThumbnailContainer] = [
+    document.querySelector<HTMLTableElement>(".courselist tbody"),
+    document.querySelector<HTMLElement>(".mycourses-body .section"),
+  ]
```

Comment:
```suggestion
  const coursesListContainer =
    document.querySelector<HTMLElement>(".courselist tbody")
  const coursesThumbnailContainer = document.querySelector<HTMLElement>(
    ".mycourses-body .section"
  )
```

I don't see any positive reason for using destructuring assignments here. Do you hava some?